Demo/Demo/SwiftUIComponents/PayPalWebViews/PayPalWebOrderCompletionResultView.swift

```diff
@@ -39,6 +39,14 @@ struct PayPalWebOrderCompletionResultView: View {
                 LeadingText("Email", weight: .bold)
                 LeadingText("\(email)")
             }
+            if let vaultID = orderResponse.paymentSource?.paypal?.attributes?.vault.id {
+                LeadingText("Vault ID / Payment Token", weight: .bold)
+                LeadingText("\(vaultID)")
+            }
```

Comment:
General question: is there an agreed upon name that JS calls this in our public docs? I see we use both vault ID and payment token here. We should use whichever name is used in our public docs.

Reply:
I think for the customer point of view, we call it payment token , but we reference it as vault ID in the APIs. 
I just referenced vault Id on the label to clarify that they are the same.